### PR TITLE
Enabling Random name of PV/PVC/Independent App and also pre-check to delete all resources if they are present

### PR DESCRIPTION
**Pull request by dbhatnagar-px**
This PR takes care of any duplicate PV/PVC/Independent app that may already been running in the system. Also just in case they are running, this PR makes sure to delete them before proceeding ahead with the TC

### CODE REVIEW

**Review by gauraa2 — APPROVED** [review]
LGTM

File: drivers/pds/lib/utils.go [hunk 1]
```diff
@@ -1310,7 +1362,7 @@ func CreateIndependentPV(name string) (*corev1.PersistentVolume, error) {
 			},
 		},
 	}
-	pv, err := k8sCore.CreatePersistentVolume(pv)
+	pv, err = k8sCore.CreatePersistentVolume(pv)
```
Comment (lsrinivas-pure) [inline drivers/pds/lib/utils.go:1365]: can u delete the below error logging?

Comment (dbhatnagar-px) [reply drivers/pds/lib/utils.go:1365]: Done

File: drivers/pds/lib/utils.go [hunk 2]
```diff
@@ -1286,8 +1292,54 @@ func RegisterClusterToControlPlane(controlPlaneUrl, tenantId, clusterType string
...
+// Check if PV and associated PVC is still present. If yes then delete both of them
+func CheckAndDeleteIndependentPV(name string) error {
+	pv_check, err := k8sCore.GetPersistentVolume(name)
+	if err != nil {
+		if strings.Contains(err.Error(), "not found") {
+			return nil
+		}
+		return err
+	}
+	log.InfoD("Stranded PV Found. Going ahead to delete this PV and associated entities")
```
[the remark this answers is not in the document]
Comment (dbhatnagar-px) [reply drivers/pds/lib/utils.go]: Done

**Pull request merged**